go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.2
 	github.com/openshift/api v0.0.0-20220124143425-d74727069f6f
+	github.com/openshift/build-machinery-go v0.0.0-20220121085309-f94edc2d6874
```

Comment:
Wait for https://github.com/openshift/build-machinery-go/pull/66 to merge, then I will upgrade it to the latest version